src/app/organizations/collection/collection.component.html

```diff
@@ -72,68 +66,93 @@ <h1>
             <mat-icon>edit</mat-icon>
           </button>
         </mat-card-actions>
+      </div>
+      <mat-card class="alert alert-info" *ngIf="collection?.entries?.length == 0">
+        <mat-icon>info</mat-icon> This collection has no associated entries. To add an entry, navigate to the public page of the tool or
+        workflow you want to add and select `Add to collection`
       </mat-card>
-
-      <div fxLayout="row" fxLayout.lt-md="column" fxLayoutGap="15px" class="my-3">
+      <div fxLayout="row" fxLayout.lt-md="column" fxLayoutGap="30px" class="my-3">
         <div fxFlex fxGrow="1" fxFlex.lt-md="100%">
-          <mat-card class="alert alert-info" *ngIf="collection?.entries?.length == 0">
-            <mat-icon>info</mat-icon> This collection has no associated entries. To add an entry, navigate to the public page of the tool or
-            workflow you want to add and select `Add to collection`
-          </mat-card>
-
           <div fxLayout="column" fxLayoutGap="15px">
+            <h3>Workflows & Tools</h3>
             <div data-cy="collectionEntry" *ngFor="let entry of collection?.entries">
-              <mat-card class="entryCard">
-                <mat-card-header>
-                  <mat-card-title>
-                    <h4>
-                      <a
-                        [routerLink]="['/' + entry.entryType + 's/', entry.entryPath + (entry.versionName ? ':' + entry.versionName : '')]"
-                      >
+              <mat-card class="mat-elevation-z3">
+                <a
+                  [routerLink]="[
+                    '/' + (entry.entryType === 'tool' ? 'container' : 'workflow') + 's/',
+                    entry.entryPath + (entry.versionName ? ':' + entry.versionName : '')
+                  ]"
+                >
+                  <div fxLayout="row" fxLayoutAlign="space-between">
+                    <div fxLayout="row" fxLayoutAlign="flex-start" fxLayoutGap="1rem">
+                      <img *ngIf="entry.entryType === 'workflow'" src="../../../assets/svg/workflow.svg" alt="workflow icon" />
+                      <img *ngIf="entry.entryType === 'tool'" src="../../../assets/svg/tool.svg" alt="tools icon" />
```

Comment:
Real nitpicky, but you have "workflow icon", singular, and "tools icon", plural.